arch/src/x86_64/mod.rs

```diff
@@ -127,7 +129,7 @@ pub fn configure_system(
     cmdline_size: usize,
     num_cpus: u8,
     setup_hdr: Option<setup_header>,
-    serial_enabled: bool,
+    _serial_enabled: bool,
```

Comment:
Why did you add an underscore here?

Reply:
Otherwise the "no default" build on travis will fail because that parameter is only used in the acpi feature.